### PR TITLE
[TEVA-1600] Update job alert confirmation link

### PR DESCRIPTION
**Pull request by cpjmcquillan**
## Jira ticket URL
https://dfedigital.atlassian.net/browse/TEVA-1600

## Changes in this PR
- Render different link on job alert confirmation pages when jobseeker accounts are enabled
- Update job alert from search system spec

### CODE REVIEW

File: spec/system/jobseekers_can_create_a_job_alert_from_a_search_spec.rb [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+      activities = PublicActivity::Activity.all
+      keys = activities.pluck(:key)
+      expect(keys).to include("subscription.alert.new")
+      expect(keys).to include("subscription.daily_alert.create")
+    end
+  end
+
+  def and_click_job_alert_link
+    if page.has_css?("#job-alert-link")
+      click_on("Receive a job alert")
```
Comment (cesidio) [inline spec/system/jobseekers_can_create_a_job_alert_from_a_search_spec.rb:128]: Should we be consistent and use I18n keys here too?

**Review by cesidio — APPROVED** [review]

File: spec/system/jobseekers_can_create_a_job_alert_from_a_search_spec.rb (hunk 1 above)
Comment (cpjmcquillan) [reply spec/system/jobseekers_can_create_a_job_alert_from_a_search_spec.rb:128]: I did think about it, but also didn't want to change too much things I was just copying across. In specs I find sometimes there is a balance to be had between something that is easy to read and understand (the string itself that I click on) and consistency and not repeating ourselves (using the i18n translation). Let me smash my head about it for a bit 😂

Comment (cpjmcquillan) [reply spec/system/jobseekers_can_create_a_job_alert_from_a_search_spec.rb]: I did it (the translation needed updating anyway to something a bit cleaner)

**Comment by martin-bangoura** [comment]
Review app deployed to <https://teaching-vacancies-review-pr-2577.london.cloudapps.digital>

**Pull request merged**